tests/integration_tests/utils.py

```diff
@@ -683,3 +686,106 @@ def train_with_backend(
     finally:
         # Remove results/intermediate data saved to disk
         shutil.rmtree(output_dir, ignore_errors=True)
+
+
+# creates synthetic data and update feature definitions
```

Comment:
can we add python style comments here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Updated per comment.